__tests__/app.test.js

```diff
@@ -0,0 +1,48 @@
+const app = require("../app");
+const request = require("supertest");
+const db = require("../db/connection");
+const seed = require("../db/seeds/seed");
+const {
```

Comment:
If you don't destructure here, you can require it in as data and just pass that to the seed function as it gets destructured in the seed function anyway.